README.md

```diff
@@ -114,49 +114,105 @@ The Vonage Video API Reference App for React is currently supported on the lates
 
 ## Running Locally
 
-### Config
+### 1. Ensure You Have a Vonage Account
 
- In project directory, create the environment variables for the project.
+To get started, make sure you have a Vonage account. You can create one at the [Vonage API Dashboard](https://dashboard.vonage.com/applications). The sign-up process is straightforward.
 
-```console
-cp backend/.env.example backend/.env
-```
-[Click here](backend/.env.example) to learn more about config variables used in the backend.
+### 2. Create an Application in the Dashboard
 
-Add your Vonage Video API credentials to the newly created .env file.
+Once logged in, navigate to the [Applications page](https://dashboard.vonage.com/applications) via the main dashboard menu:
 
-```console
-cp frontend/.env.example frontend/.env
+<details open>
```

Comment:
I think this one and others should be `closed` by default and not `open` to be consistent and to make the page too crowded with the screenshots